### PR TITLE
feat: git-operator release-0.13.4

### PR DESCRIPTION
**Pull request by rajgitnutanix**
**What problem does this PR solve?**:
git operator release v0.13.4

**Which issue(s) does this PR fix?**:
https://jira.nutanix.com/browse/NCN-104608

### CODE REVIEW

File: services/git-operator/0.1.2/git-operator-manifests/all.yaml
Comment (mhrabovcin) [inline services/git-operator/0.1.2/git-operator-manifests/all.yaml:1]: The version directory name needs to be changed to `0.1.3`.

Comment (rajgitnutanix) [reply services/git-operator/0.1.2/git-operator-manifests/all.yaml:1]: ok Martin will change as suggested

**Comment by mhrabovcin** [comment]
@rajgitnutanix please run:

```
devbox run just git-operator-fetch-manifests
```

To have a complete bump.

**Comment by rajgitnutanix** [comment]
> @rajgitnutanix please run:
> 
> ```
> devbox run just git-operator-fetch-manifests
> ```
> 
> To have a complete bump.

executed the same .as suggested Martin

**Review by mhrabovcin — APPROVED** [review]

**Pull request merged**